test/tests/userobjects/setup_interface_count/tests

```diff
@@ -0,0 +1,60 @@
+# The following tests are designed to count how often the setup methods are called (initialSetup, etc.),
+# which requires that the threading model and number of threads be set to known values.
+#
+# Also, GeneralUserObject must also be restricted for a certain MPI because there methods will be called
+# on each processor, since they are not distributed in any fashion (e.g., on elements)
+
+[Tests]
+  [./GeneralUserObject]
+    type = CSVDiff
+    input = general.i
+    csvdiff = general_out.csv
+    min_threads = 2
+    max_threads = 2
+    min_parallel = 2
+    max_parallel = 2
+    tbb = false
+    openmp = false
+    pthreads = true
```

Comment:
Actually that's definitely not true:
libMesh normally picks up `pthreads` all the time on our boxes but it has a lower precedent than `tbb`. Furthermore, you can actually turn on all three models and it'll run with `tbb`. You don't have to explicitly turn off the models with lower precedence. See why I don't like these flags? :smile: